cwf/gateway/integ_tests/gx_reauth_test.go

```diff
@@ -445,10 +440,8 @@ func TestGxReAuthQuotaRefill(t *testing.T) {
 		},
 	)
 	assert.NoError(t, err)
-	tr.WaitForReAuthToProcess()
+	assert.Eventually(t, tr.WaitForReAuthToProcess(raa, imsi), time.Minute, 2*time.Second)
```

Comment:
Does it have to be such a big vaule? If they all fail, we will have to wait for a minute in all the cases. 

If CI takes like 10 s to execute a reauth then maybe we do have a problem.

Reply:
That's a good point. Hopefully we fail far fewer times than we pass. Also we could monitor the test completion times and change this over time.